templates/uploadFromOpenCfp/uploadFromOpenCfp.html.twig

```diff
@@ -0,0 +1,40 @@
+{% extends 'base.html.twig' %}
+
+{% block title %}Import Speakers with Talks{% endblock %}
+
+{% block content %}
+    <div class="row">
+        <div class="col-12">
+            <h1>Import Speakers with Talks</h1>
+        </div>
+    </div>
+
+    {{ form_start(form) }}
+    <div class="row">
+        <div class="col-12">
+            {{ form_widget(form) }}
+        </div>
+    </div>
+
+    <div class="row mb-3">
+        <div class="col-6">
+            <a href="{{ path('speaker_index') }}" class="btn btn-light">Back to list</a>
+        </div>
+        <div class="col-6 text-right">
+            <button type="submit" class="btn btn-success">Import</button>
+        </div>
+    </div>
+    {{ form_end(form) }}
+
+    <script>
+        document.querySelector("input[type=file]").addEventListener('change', function (element) {
+            let target = element.target;
+
+            const value = target.value.replace("C:\\fakepath\\", "");
```

Comment:
not sure about this script, specific to windows

Reply:
the C:\\fakepath concern Chrome, this is a security to mask the real path of the file, but that was not the issue here, I fixed the JS in this part